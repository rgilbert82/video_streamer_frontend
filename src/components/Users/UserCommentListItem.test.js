import React                   from 'react';
import { shallow }             from 'enzyme';
import { UserCommentListItem } from '.';

describe(UserCommentListItem, () => {
  const comment = {
    message: 'hello',
    published_at: '2018'
  };
  let component;

  beforeEach(() => {
    component = shallow(<UserCommentListItem comment={comment} />);
  });

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
});
