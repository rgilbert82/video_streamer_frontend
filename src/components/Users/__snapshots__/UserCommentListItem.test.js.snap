// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class UserCommentListItem extends _react2.default.Component {
  render() {
    return (
      _react2.default.createElement('div', { __source: { fileName: _jsxFileName, lineNumber: 6 }, __self: this },
        _react2.default.createElement('p', { __source: { fileName: _jsxFileName, lineNumber: 7 }, __self: this }, this.props.comment.message),
        _react2.default.createElement('small', { __source: { fileName: _jsxFileName, lineNumber: 8 }, __self: this }, this.props.comment.published_at)));


  }} renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserCommentListItem
    comment={
        Object {
            "message": "hello",
            "published_at": "2018",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          hello
</p>,
        <small>
          2018
</small>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "hello",
        },
        "ref": null,
        "rendered": "hello",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "2018",
        },
        "ref": null,
        "rendered": "2018",
        "type": "small",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            hello
</p>,
          <small>
            2018
</small>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "hello",
          },
          "ref": null,
          "rendered": "hello",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "2018",
          },
          "ref": null,
          "rendered": "2018",
          "type": "small",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
